modules/games.py

```diff
@@ -198,13 +198,15 @@ def startGame(self, message: core.BotMessage) -> None:
         if not message.sender.can("hostgame", message.room): return message.respond("Permission denied.")
         isTournament = message.arguments[0].strip(config.commandCharacter) in ['tour', 'tournament']
         if isTournament and len(message.arguments) < 2: return message.respond(
-            f"Usage: ``{config.commandCharacter}tournament [format]``."
+            f"Usage: ``{config.commandCharacter}tournament [format], [comma-separated custom rules]``."
         )
 
         if isTournament:
             commands = TOUR_SETUP_COMMANDS
-            tourFormat = " ".join(message.arguments[1:])
+            tourFormat = " ".join(message.arguments[1])
```

Comment:
```suggestion
            tourFormat = message.arguments[1]
```
`message.arguments[1]` is a string; you don't have to join it.